lib/google/apis/core/http_command.rb

```diff
@@ -288,15 +297,15 @@ def execute_once(client)
             request_header = header.dup
             apply_request_options(request_header)
 
-            http_res = client.request(method.to_s.upcase,
-                                      url.to_s,
-                                      query: nil,
-                                      body: body,
-                                      header: request_header,
-                                      follow_redirect: true)
-            logger.debug { http_res.status }
-            logger.debug { http_res.inspect }
-            response = process_response(http_res.status.to_i, http_res.header, http_res.body)
+            @http_res = client.request(method.to_s.upcase,
```

Comment:
Should we worry that this ivar could be held on to longer than wanted? Should the `ensure` block in `execute` set this ivar to `nil`?

Reply:
Under current usage, the HttpCommand object is short-lived. But it can't hurt to set to nil. Done.